### ISSUE TITLE
[BUG] StatsBase.Histogram does not work anymore

### ISSUE THREAD

**Issue by lmh91** [JuliaPlots/Plots.jl#3137]
## Details

I guess there is something wrong with the edges.
The bug does not occur on Plots 1.7.3.

MWE:
```julia
using Plots, StatsBase
h = fit(Histogram, rand(10))
plot(h)
```
Output:
julia> plot(h)
ERROR: Expects 6 elements in each col of y, found 5.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] _compute_xyz(::StepRangeLen{Float64,Base.TwicePrecision{Float64},Base.TwicePrecision{Float64}}, ::Array{Float64,1}, ::Nothing) at /user/.julia/packages/RecipesPipeline/ZVaHH/src/series.jl:97
 [3] macro expansion at /user/.julia/packages/RecipesPipeline/ZVaHH/src/series.jl:163 [inlined]
 [4] apply_recipe(::AbstractDict{Symbol,Any}, ::Type{RecipesPipeline.SliceIt}, ::Any, ::Any, ::Any) at /user/.julia/packages/RecipesBase/92zOw/src/RecipesBase.jl:282
 [5] _process_userrecipes!(::Any, ::Any, ::Any) at /user/.julia/packages/RecipesPipeline/ZVaHH/src/user_recipe.jl:36
 [6] recipe_pipeline!(::Any, ::Any, ::Any) at /user/.julia/packages/RecipesPipeline/ZVaHH/src/RecipesPipeline.jl:70
 [7] _plot!(::Plots.Plot, ::Any, ::Any) at /user/.julia/packages/Plots/YdauZ/src/plot.jl:172
 [8] #plot#129 at /user/.julia/packages/Plots/YdauZ/src/plot.jl:58 [inlined]
 [9] plot(::Any) at /user/.julia/packages/Plots/YdauZ/src/plot.jl:52
 [10] top-level scope at REPL[9]:1

### Backends

This bug occurs on ( insert `x` below )

Backend      | yes | no  | untested
-------------|-----|-----|---------
gr (default) |  x  |     |
pyplot       |  x  |    |
plotly       |     |     | x
plotlyjs     |     |     | x
pgfplotsx    |     |     | x
inspectdr    |     |     | x

### Versions
Plots.jl version: 1.8.1
Backend  version (`]st -m`): 
Status `~/code/JuliaScripts/test/Manifest.toml`
  [79e6a3ab] Adapt v2.3.0
  [56f22d72] Artifacts v1.3.0
  [6e34b625] Bzip2_jll v1.0.6+5
  [83423d85] Cairo_jll v1.16.0+6
  [35d6a980] ColorSchemes v3.10.1
  [3da002f7] ColorTypes v0.10.9
  [5ae59095] Colors v0.12.4
  [34da2185] Compat v3.23.0
  [d38c429a] Contour v0.5.5
  [9a962f9c] DataAPI v1.4.0
  [864edb3b] DataStructures v0.18.8
  [e2d170a0] DataValueInterfaces v1.0.0
  [5ae413db] EarCut_jll v2.1.5+1
  [2e619515] Expat_jll v2.2.7+6
  [c87230d0] FFMPEG v0.4.0
  [b22a6f82] FFMPEG_jll v4.3.1+4
  [53c48c17] FixedPointNumbers v0.8.4
  [a3f928ae] Fontconfig_jll v2.13.1+14
  [59287772] Formatting v0.4.1
  [d7e528f0] FreeType2_jll v2.10.1+5
  [559328eb] FriBidi_jll v1.0.5+6
  [0656b61e] GLFW_jll v3.3.2+1
  [28b8d3ca] GR v0.53.0
  [d2c73de3] GR_jll v0.52.0+0
  [5c1252a2] GeometryBasics v0.3.3
  [78b55507] Gettext_jll v0.20.1+7
  [7746bdde] Glib_jll v2.59.0+4
  [42e2da0e] Grisu v1.0.0
  [cd3eb016] HTTP v0.8.19
  [83e8ac13] IniFile v0.5.0
  [c8e1da08] IterTools v1.3.0
  [82899510] IteratorInterfaceExtensions v1.0.0
  [692b3bcd] JLLWrappers v1.1.3
  [682c06a0] JSON v0.21.1
  [aacddb02] JpegTurbo_jll v2.0.1+3
  [c1c5ebd0] LAME_jll v3.100.0+3
  [dd4b983a] LZO_jll v2.10.0+3
  [b964fa9f] LaTeXStrings v1.2.0
  [23fbe1c1] Latexify v0.14.3
  [dd192d2f] LibVPX_jll v1.9.0+1
  [e9f186c6] Libffi_jll v3.2.1+4
  [d4300ac3] Libgcrypt_jll v1.8.5+4
  [7e76a0d4] Libglvnd_jll v1.3.0+3
  [7add5ba3] Libgpg_error_jll v1.36.0+3
  [94ce4f54] Libiconv_jll v1.16.0+7
  [4b2f31a3] Libmount_jll v2.34.0+3
  [89763e89] Libtiff_jll v4.1.0+2
  [38a345b3] Libuuid_jll v2.34.0+7
  [1914dd2f] MacroTools v0.5.6
  [739be429] MbedTLS v1.0.3
  [c8ffd9c3] MbedTLS_jll v2.16.8+1
  [442fdcdd] Measures v0.3.1
  [e1d29d7a] Missings v0.4.4
  [77ba4419] NaNMath v0.3.4
  [e7412a2a] Ogg_jll v1.3.4+2
  [458c3c95] OpenSSL_jll v1.1.1+6
  [91d4177d] Opus_jll v1.3.1+3
  [bac558e1] OrderedCollections v1.3.2
  [2f80f16e] PCRE_jll v8.42.0+4
  [69de0a69] Parsers v1.0.12
  [30392449] Pixman_jll v0.38.4+5
  [ccf2f8ad] PlotThemes v2.0.0
  [995b91a9] PlotUtils v1.0.7
  [91a5bcdd] Plots v1.8.1
  [784f63db] Qhull_jll v2019.1.0+2
  [ede63266] Qt_jll v5.15.0+3
  [3cdcf5f2] RecipesBase v1.1.1
  [01d81517] RecipesPipeline v0.2.0
  [189a3867] Reexport v0.2.0
  [ae029012] Requires v1.1.0
  [6c6a2e73] Scratch v1.0.3
  [992d4aef] Showoff v0.3.2
  [a2af1166] SortingAlgorithms v0.3.1
  [90137ffa] StaticArrays v0.12.5
  [2913bbd2] StatsBase v0.33.2
  [09ab397b] StructArrays v0.4.4
  [3783bdb8] TableTraits v1.0.0
  [bd369af6] Tables v1.2.1
  [a2964d1f] Wayland_jll v1.17.0+4
  [2381bf8a] Wayland_protocols_jll v1.18.0+4
  [02c8fc9c] XML2_jll v2.9.10+3
  [aed1982a] XSLT_jll v1.1.33+4
  [4f6342f7] Xorg_libX11_jll v1.6.9+4
  [0c0b7dd1] Xorg_libXau_jll v1.0.9+4
  [935fb764] Xorg_libXcursor_jll v1.2.0+4
  [a3789734] Xorg_libXdmcp_jll v1.1.3+4
  [1082639a] Xorg_libXext_jll v1.3.4+4
  [d091e8ba] Xorg_libXfixes_jll v5.0.3+4
  [a51aa0fd] Xorg_libXi_jll v1.7.10+4
  [d1454406] Xorg_libXinerama_jll v1.1.4+4
  [ec84b674] Xorg_libXrandr_jll v1.5.2+4
  [ea2f1a96] Xorg_libXrender_jll v0.9.10+4
  [14d82f49] Xorg_libpthread_stubs_jll v0.1.0+3
  [c7cfdc94] Xorg_libxcb_jll v1.13.0+3
  [cc61e674] Xorg_libxkbfile_jll v1.1.0+4
  [12413925] Xorg_xcb_util_image_jll v0.4.0+0
  [2def613f] Xorg_xcb_util_jll v0.4.0+0
  [975044d2] Xorg_xcb_util_keysyms_jll v0.4.0+0
  [0d47668e] Xorg_xcb_util_renderutil_jll v0.3.9+0
  [c22f9ab0] Xorg_xcb_util_wm_jll v0.4.1+0
  [35661453] Xorg_xkbcomp_jll v1.4.2+4
  [33bec58e] Xorg_xkeyboard_config_jll v2.27.0+4
  [c5fb5394] Xorg_xtrans_jll v1.4.0+3
  [83775a58] Zlib_jll v1.2.11+18
  [3161d3a3] Zstd_jll v1.4.5+2
  [0ac62f75] libass_jll v0.14.0+4
  [f638f0a6] libfdk_aac_jll v0.1.6+4
  [b53b4c65] libpng_jll v1.6.37+6
  [f27f6e37] libvorbis_jll v1.3.6+6
  [1270edf5] x264_jll v2020.7.14+2
  [dfaa095f] x265_jll v3.0.0+3
  [d8fb68d0] xkbcommon_jll v0.9.1+5
  [2a0f44e3] Base64
  [ade2ca70] Dates
  [8bb1440f] DelimitedFiles
  [8ba89e20] Distributed
  [b77e0a4c] InteractiveUtils
  [76f85450] LibGit2
  [8f399da3] Libdl
  [37e2e46d] LinearAlgebra
  [56ddb016] Logging
  [d6f4376e] Markdown
  [a63ad114] Mmap
  [44cfe95a] Pkg
  [de0858da] Printf
  [3fa0cd96] REPL
  [9a3f8284] Random
  [ea8e919c] SHA
  [9e88b42a] Serialization
  [1a1011a3] SharedArrays
  [6462fe0b] Sockets
  [2f01184e] SparseArrays
  [10745b16] Statistics
  [8dfed614] Test
  [cf7118a7] UUIDs
  [4ec0a83e] Unicode

Output of `versioninfo()`:
Julia Version 1.5.2
Commit 539f3ce943 (2020-09-23 23:17 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, skylake)
Environment:
  JULIA_DEPOT_PATH = /user/.julia:
  JULIA_NUM_THREADS = 4
  JULIA_EDITOR = code -n -w
  JULIA_PKGDIR = /user/.julia
  JULIA_SCRIPTS_DIR = /home/lhauert/code/JuliaScripts
  JULIA_PKG_DEVDIR = /home/lhauert/code/JuliaPackages
  JULIA_CXX_RTTI = 1

**Comment by lmh91** [comment]
Might be related to #3136

**Comment by oschulz** [comment]
`plot(x, y, seriestype = :stepbins)` is broken too now, with the same error. Strangely, `stephist` (that uses `:stepbins`) still works.

**Comment by daschw** [comment]
This is fixed for RecipesPipeline 0.2.1 thanks to @oschulz in https://github.com/JuliaPlots/RecipesPipeline.jl/pull/71.
